lib/galaxy/datatypes/binary.py

```diff
@@ -200,16 +201,16 @@ def display_peek(self, dataset):
             return "Compressed binary file (%s)" % (nice_size(dataset.get_size()))
     
     def sniff(self, filename):
-        # If the zip file contains multiple files then return true
-        zf = zipfile.ZipFile(filename)
-        zf_files = zf.infolist()
-        # Filter the directories within the file list
-        zf_files = [x.filename for x in zf_files if x.file_size > 0]
-        # Filter hidden file within the file list
-        hidden_file_patterns = ["__MACOSX",".DS_Store","^\."]
-        regex = re.compile('|'.join(hidden_file_patterns))
-        zf_files = [x for x in zf_files if x not in list(filter(regex.search, zf_files))]
-        return len(zf_files) > 1
+        UNDESIRED_FILES = {'.*__MACOSX.*', '.*\.DS_Store.*'}
```

Comment:
Do you have examples of these ? At least the `.DS_Store` files never have any suffix or prefix.

Reply:
You're right.
```
dossier1/.DS_Store
__MACOSX/dossier1/._.DS_Store
```
I can simplify with: 
```
UNDESIRED_FILES = {'.*__MACOSX.*', '.*\.DS_Store'}
```